### DISCUSSION TITLE
🙏🙏🙏WCFM Ultimate - Vendor Badges compatibility🔥

### DISCUSSION THREAD

**Discussion by Yan-Aleman** [inspireui/support#5344] [New Feature/Ideas Requests] [answered]
Vendors purchase tariff plans and receive "badges" depending on the tariff plan. Please add "badges". And also the "store description" vendors in wcfm..

**Comment by Yan-Aleman** [comment, marked as answer]
@minhcasi @simonpham

**Comment by revoru** [comment]
Will the vendor badges be supported?
